conda_env/specs/binstar.py

```diff
@@ -81,15 +82,24 @@ def environment(self):
         :raises: EnvironmentFileNotDownloaded
         """
         if self._environment is None:
+
             versions = [{
                 'normalized': normalized_version(d['version']),
                 'original': d['version']} for d in self.file_data]
-            latest_version = max(versions, key=lambda x: x['normalized'])['original']
-            file_data = [data for data in self.package['files'] if data['version'] == latest_version]
-            req = self.binstar.download(self.username, self.packagename, latest_version, file_data[0]['basename'])
+
+            if self.version:
+                print("We have a version! {}".format(self.version))
```

Comment:
This seems like a debugging message.  Can it be cut or turned into a log message instead?